packages/client/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,100 @@
+import styled from 'styled-components';
+import React, { useEffect, useState } from 'react';
+
+type InputProps = {
```

Comment:
Давай сделаем так:

```suggestion
type InputProps = {
  type: string;
  placeholder: string;
  value?: string;
  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
  validationText?: string;
};

const Input = (props: InputProps) => {
```

И перенесем эти типы в отдельный файл types.ts

Reply:
Все что касается цвета и т.д. мы будем брать из темы, не прокидывая напрямую

Reply:
Сделано!